packages/core/src/__tests__/embrace.test.ts

```diff
@@ -303,18 +303,53 @@ describe("Personas Tests", () => {
 });
 
 describe("Custom Views Tests", () => {
-  const testView = "View";
-
   test("startView", async () => {
-    await startView(testView);
-    // TODO uncomment the expect once the method is implemented
-    // expect(mockStartView).toHaveBeenCalledWith(testView);
+    const mock = jest.fn();
+    jest.mock(
```

Comment:
should avoid mocking these inline, there's a single NativeModules mock at the top that can be used, see https://github.com/embrace-io/embrace-react-native-sdk/pull/67